src/DatePicker/DatePicker.js

```diff
@@ -23,6 +22,12 @@ class DatePicker extends Component {
         this.calendarRef = React.createRef();
     }
 
+    componentDidMount() {
+        if (!this.props.disableStyles) {
+            require('fundamental-styles/dist/popover.css');
```

Comment:
This whole section can be removed if https://github.com/SAP/fundamental-react/pull/858 is merged first. That rebuilds this with a popover so we get those styles from popover component.